### DISCUSSION TITLE
What is this "idle" part of the card?

### DISCUSSION THREAD

**Discussion by kylewilliams321** [moralmunky/Home-Assistant-Mail-And-Packages#552] [Q&A]
Hopefully this image comes through.
Out of curiosity what is this idle message about?

![image](https://user-images.githubusercontent.com/39958233/139450187-b9beda79-d4e0-4a24-ad23-310f158230a2.png)

**Comment by firstof9** [comment]
Interesting, let's ping @moralmunky for this one. 😅

**Comment by kylewilliams321** [comment]
My guess is its this line: packages_delivered: sensor.mail_amazon_packages_delievered, but I can't tell for sure but here it is in developer tools:
![image](https://user-images.githubusercontent.com/39958233/139452422-6432e033-5002-4181-ae4a-e8aa1c067559.png)

**Reply by moralmunky** [reply]
You have the deliveries_message set to the camera entity so it is showing that the camera is idle.

The deliveries_message is for a plain language message describing what deliveries are happening that day. It is a sensor you need to create yourself. There is an example [here](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki/Mail-Summary-Message#text-summary-examples).

**Reply by firstof9** [reply]
Ha, good eye, I didn't even notice lol

**Comment by kylewilliams321** [comment]
Thank you. Will look into your link.

**Comment by kylewilliams321** [comment]
Nailed it. THANKS!
![image](https://user-images.githubusercontent.com/39958233/139477595-394b5869-9bc2-48ac-b48a-59fdbfa3a4f4.png)
